EventFilter/L1TRawToDigi/src/implementations_stage2/MPUnpacker_0x1001000b.cc

```diff
@@ -61,27 +77,27 @@ namespace stage2 {
      l1t::EtSum et = l1t::EtSum();
 
      switch(block.header().getID()){
-     case 123:
+     case BLK_TOT_POS:
        et.setType(l1t::EtSum::kTotalEt);
        et.setHwPt( static_cast<int32_t>( uint32_t(raw_data & 0xFFFFFF) << 16 ) >> 16 );
        break;
-     case 121:
+     case BLK_X_POS:
        et.setType(l1t::EtSum::kTotalEtx);
        et.setHwPt( static_cast<int32_t>( uint32_t(raw_data) ) );
```

Comment:
There is not need to cast `raw_data` to  `uint32_t`. It's already `uint32_t`: `uint32_t raw_data = block.payload()[fet];`

Same applies to some below this comment.